pyproject.toml

```diff
@@ -44,6 +44,7 @@ pytest-rerunfailures = "^10.1"
 num2tex = "^0.8"
 mike = "^1.1.2"
 pydocstyle = "^6.1.1"
+ipython-genutils = "^0.2.0"
```

Comment:
Could we try bumping to `mkdocs-jupyter = "^0.21.0"` instead? `ipython-genutils` sound like being a sub-dep of this package which we pinned to a rather old version. Updating might solve this. 

If that doesn't work out in the first try, I'm also happy with this workaround.